releng/freemarker-ide.target

```diff
@@ -23,14 +23,11 @@
       <unit id="org.junit" version="0.0.0"/>
       <unit id="org.eclipse.ui.workbench" version="0.0.0"/>
       <unit id="org.eclipse.ui.ide.application" version="0.0.0"/>
-      <repository location="https://download.eclipse.org/releases/2021-06"/>
-    </location>
-    <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="true" type="InstallableUnit">
-      <repository location="https://download.eclipse.org/eclipse/updates/4.20"/>
+      <repository location="http://download.jboss.org/jbosstools/targetplatforms/jbosstoolstarget/4.71.1.Final/REPO"/>
```

Comment:
2019-06 is https://download.jboss.org/jbosstools/targetplatforms/jbosstoolstarget/4.12.0.Final-SNAPSHOT/REPO/, I think.